ui/raidboss/data/06-ew/raid/p3s.ts

```diff
@@ -229,6 +238,90 @@ const triggerSet: TriggerSet<Data> = {
         north: Outputs.north,
       },
     },
+    {
+      id: 'P3S Sunbird Tether Collector',
+      type: 'Tether',
+      // 0039 when pink, 0001 when stretched purple.
+      // TODO: in general, it seems like the tethers are picked to start unstretched,
+      // but plausibly you could create a scenario where one starts stretched?
+      netRegex: NetRegexes.tether({ id: ['0039', '0001'] }),
```

Comment:
You could narrow this to source: 'Sunbird' later 'Sunshadow's spawn that also create a 0001 tether.

Doesn't make any functional difference though.

Reply:
Hmm, I don't think I can narrow it, because it could be between players too.  I think right now if a player somehow doesn't get a tether during add phase, then it'll fall into the "no bird" error output and not print anything on screen.  I can make a note that it might happen.

Reply:
Ohhhh, you're right, thanks for explaining.  Updated.